### PR TITLE
Experimenal - Github Action for exporting PDFs

### PR DESCRIPTION
**Pull request by adeeena**
Remarks:

* The exported pdfs go to another branch (`export-pdfs`), for manual verification (before it goes directly on public), also to check whether a github action, which performs push on itself will re-trigger another action, possibly causing an infinite build loop.

* I added links to the PDFs to each language, however apart from Hungarian I had to rely on Google Translate for the labels. Please verify (DE @TheBlackAlpha48 , PL @ErdaradunGaztea , CN-ZH @rqnj )

* Reading the action yaml, you might find that I'm working on a translation for another language.

* _Why the checks failing?_ the .yml file tries to do the push on the {current user}/GenderDysphoria.fyi, which is `adeeena` in this PR.

### CODE REVIEW

File: public/pl/_menu.hbs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <a href="/pl/" class="top-nav-item dropdown-toggle" id="nav-gdb" data-toggle="dropdown" data-flip="false" aria-haspopup="true" aria-expanded="false">Biblia Dysforii Płciowej</a>
 <div class="dropdown-menu" aria-labelledby="nav-gdb">
   <a href="/pl/do-druku"                 class="{{#is meta.url '/pl/do-druku'                 }}active {{/is}}dropdown-item">Zobacz wszystko</a>
+  <a href="/gdb-pl.pdf"                     class="dropdown-item">Ściągnij PDF {{icon 'pdf' size="1em" style="vertical-align:middle;margin-bottom:3px;"}} </a>
```
Comment (ErdaradunGaztea) [inline public/pl/_menu.hbs:5]: "Ściągnij PDF" is perfectly fine, but "Pobierz PDF" would sound even more natural.

Comment (adeeena) [reply public/pl/_menu.hbs]: Thanks! I changed the label.

**Review by ErdaradunGaztea — APPROVED** [review]

**Review by TheBlackAlpha48 — APPROVED** [review]

**Comment by Twipped** [comment]
Sorry for not seeing this sooner. New job uses github, which means notifications from this repo kinda vanish in the flood of work related PRs.

@adeeena `secrets.GITHUB_TOKEN` is only scoped for the repo running the workflow. You don't have commit rights to this repo, so it's failing.

Rather than commit them, how about having it produce an artifact? I'm planning to construct my existing workflow into a proper deployment script that generates an artifact, so I could tack this on as an extra job and combine the two artifacts for push to S3.

**Comment by Twipped** [comment]
There we go. I'll iterate on it further, but this is an excellent start. Thanks!

**Pull request merged**